transport/websocketpeer.go

```diff
@@ -69,14 +70,35 @@ const (
 
 type DialFunc func(network, addr string) (net.Conn, error)
 
+// ConnectWebsocketPeer calls ConnectWebsocketPeerContext without a Dial context
+func ConnectWebsocketPeer(
+	routerURL string,
+	serialization serialize.Serialization,
+	tlsConfig *tls.Config,
+	dial DialFunc,
+	logger stdlog.StdLog,
+	wsCfg *WebsocketConfig) (wamp.Peer, error) {
+	return ConnectWebsocketPeerContext(nil, routerURL, serialization, tlsConfig, dial, logger, wsCfg)
```

Comment:
Better to call with first argument as `context.Background()`.  Reason explained below.